### DISCUSSION TITLE
Where's v2ray config file stored?

### DISCUSSION THREAD

**Discussion by yasmoradi** [FranzKafkaYu/x-ui#226] [Q&A]
你好

As you might have seen on `xui/setting` page, there is a text as follows:

`The final config will be generated based on this template,need restart panel to take effect`

Where can I see the content of the generated file?

Thanks in advance

**Comment by wayneace** [comment]
for more detailed usages,plz see [WIKI](https://github.com/FranzKafkaYu/x-ui/wiki)
